user-docs/documentation/docs/getting-started/installation.md

```diff
@@ -4,19 +4,19 @@ sidebar_position: 1
 
 # Installation
 
-In this guide we'll see how to generate a Swarmion project from scratch. If you're looking how to migrate your existing project to Swarmion, please check out [the migration docs](../migrating)
+In this guide we'll see how to generate a Swarmion project from scratch. If you're looking how to migrate your existing project to Swarmion, please check out [the migration docs](../migrating).
+
 _Please note: this section supposes that you already have an active AWS account._
 
 ## Requirements
 
-- node v14: [install docs](https://nodejs.org/en/download/)
-- nvm: [install docs](https://github.com/nvm-sh/nvm)
+- nvm: [install docs](https://github.com/nvm-sh/nvm#installing-and-updating)
 - yarn: [install docs](https://yarnpkg.com/getting-started/install)
 - AWS CLI: [install docs](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
 
-## Generate your project from Swarmion template
+## Create a new project from the Swarmion template
 
-We'll start by generating a sample proj
+Head over to the [Swarmion template](https://github.com/swarmion/template) on GitHub. Click the button _Use this template_ and follow the instructions.
```

Comment:
Can you add a screenshot of the button in the Github interface?

Reply:
Fixed